control/server.py

```diff
@@ -41,6 +42,17 @@ def callable():
 
     return callable
 
+def wait_for_file(file_path, timeout=None, interval=1):
+    start_time = time.time()
+
+    while True:
+        if os.path.exists(file_path):
+            return
```

Comment:
I don't remember if we'll have to enforce another condition, but this `wait_till_<something>()` methods are quite common, so we could just expose it as a generic checker:
```suggestion
def wait_for_condition(condition_method: Callable[[], bool], error_message=None, timeout=None, interval=1):
    start_time = time.time()

    while True:
        if condition_method:
            return
...

wait_for_condition(lambda: os.path.exists(spdk_file), ...)              
```